docs/tutorials/development.md

```diff
@@ -1,3 +1,47 @@
+## Local Setup
+
+### Build Skipper Binary
+
+Clone repository and compile with [Go](https://golang.org/dl).
+
+```sh
+% git clone https://github.com/zalando/skipper.git
+% cd skipper
+% make skipper
```

Comment:
Shell prompt makes it a bit harder to copy paste the command. Also below there is no shell prompt. Should we remove it maybe?